bmwqemu.pm

```diff
@@ -305,6 +305,23 @@ sub save_json_file {
     return rename("$fn.new", $fn);
 }
 
+=head2 save_yaml_file
+
+  save_yaml_file($yaml, $fn);
+
+Load YAML string and save the data as YAML into the given filename.
+Also validate YAML string.
+
+=cut
+
+sub save_yaml_file {
+    my ($yaml, $fn) = @_;
```

Comment:
We are not being charged per character :)

```suggestion
    my ($yaml, $filename) = @_;
```

Reply:
Well, I got inspired by `save_json_file`, but sure I'll update it :).